sdks/java/core/build.gradle

```diff
@@ -49,11 +49,11 @@ interface for processing virtually any size data. This
 artifact includes entire Apache Beam Java SDK."""
 
 processResources {
+  inputs.property('version', version)
+  inputs.property('sdk_version', sdk_version)
   filter org.apache.tools.ant.filters.ReplaceTokens, tokens: [
     'pom.version': version,
     'pom.sdk_version': sdk_version,
-    'pom.docker_image_default_repo_root': docker_image_default_repo_root,
```

Comment:
We still use these here: https://github.com/apache/beam/blob/0c01636fc8610414859d946cb93eabc904123fc8/sdks/java/core/src/main/java/org/apache/beam/sdk/util/ReleaseInfo.java#L107

Can we make `docker_image_default_repo_root` and `docker_image_default_repo_prefix` inputs as well?

Reply:
I can see that `docker_image_default_repo_root` is still used in this codebase and this file, but it isn't used as a replacement token in the processResource task from what I can see. Perhaps I am missing something though. Can you explain where the `docker_image_default_repo_root` is replaced or how it is used?

Reply:
> I can see that `docker_image_default_repo_root` is still used in this codebase and this file, but it isn't used as a replacement token in the processResource task from what I can see. Perhaps I am missing something though. Can you explain where the `docker_image_default_repo_root` is replaced or how it is used?

Oh, you're correct. That's strange, it looks like ReleaseInfo.java looks for `docker_image_default_repo_root` and `docker_image_default_repo_prefix` in sdk.properties, but it is never written because the sdk.properties template is missing placeholders for those fields. That seems like a bug to me. We should add those fields to the sdk.properties template.

Reply:
Good point. I added them to the sdk.properties and as inputs to the task.